pages/image-converter.tsx

```diff
@@ -0,0 +1,195 @@
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Container from '@mui/material/Container';
+import FormControl from '@mui/material/FormControl';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import FormLabel from '@mui/material/FormLabel';
+import Grid from '@mui/material/Grid';
+import Radio from '@mui/material/Radio';
+import RadioGroup from '@mui/material/RadioGroup';
+import Slider from '@mui/material/Slider';
+import { styled } from '@mui/material/styles';
+import Typography from '@mui/material/Typography';
+import React, { useState } from 'react';
+
+import Heading from '../components/Heading';
+import Layout from '../components/Layout';
+
+enum FileType {
+  PNG = 'image/png',
+  JPG = 'image/jpeg',
+  WEBP = 'image/webp',
+}
+
+const FILE_TYPE_EXTENSIONS = {
+  [FileType.PNG]: '.png',
+  [FileType.JPG]: '.jpg',
+  [FileType.WEBP]: '.webp',
+};
+
+const FILE_TYPE_OPTIONS = [
```

Comment:
No avif option, maybe browser support isn't quite there yet, if that's the case there's nothing we can do right now.

Reply:
Yes, unfortunately Chrome/Firefox still don't support AVIF in `canvas` 😢

Reply:
No worries, I'll make a new issue to solve that when it is supported